tests/house_tests.py

```diff
@@ -112,8 +108,6 @@ def test_profession_either_parent(self):
         house2.save()
         houses = _filter_houses_by_form(form_data_filter_tests(), House.objects.all())
         assert len(houses) == 2
-        assert House.objects.get(parent_profession_1=Job.TEACHER) in houses
-        assert House.objects.get(parent_profession_2=Job.TEACHER) in houses
```

Comment:
Don't we want the parent_profession_1 and parent_profession_2 to be tested?

Reply:
My mistake, I accidentally deleted it
I returned the code to the way it was